specifyweb/frontend/js_src/lib/components/DataModel/specifyField.ts

```diff
@@ -131,19 +131,19 @@ export abstract class FieldBase {
     this.isReadOnly =
       globalFieldOverride === 'readOnly' || fieldDefinition.readOnly === true;
 
+    this.localization =
```

Comment:
could you please add the test case for this in specifyField.test.ts?

https://github.com/specify/specify7/blob/0257f82efa8b5d5631a0b3245d6959a8ba55d811/specifyweb/frontend/js_src/lib/components/DataModel/__tests__/specifyField.test.ts#L48-L61